[Thu Dec 09 01:03:30.050994 2021] [php:error] [pid 33485] [client 127.0.0.1:51374] PHP Fatal error:  Uncaught Error: Class "API\\models\\User" not found in /home/ninjainside/Projects/LOCAL/login-users/index.php:10\nStack trace:\n#0 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/index.php on line 10
[Thu Dec 09 01:03:30.751192 2021] [php:error] [pid 33486] [client 127.0.0.1:51376] PHP Fatal error:  Uncaught Error: Class "API\\models\\User" not found in /home/ninjainside/Projects/LOCAL/login-users/index.php:10\nStack trace:\n#0 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/index.php on line 10
[Thu Dec 09 01:09:52.949687 2021] [php:warn] [pid 33486] [client 127.0.0.1:51468] PHP Warning:  Undefined array key 3 in /home/ninjainside/Projects/LOCAL/login-users/index.php on line 22
[Thu Dec 09 02:01:31.661980 2021] [php:error] [pid 33482] [client 127.0.0.1:52050] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php:27\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(27): array_filter()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\user\\reg()\n#2 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php on line 27
[Thu Dec 09 08:12:47.481622 2021] [php:warn] [pid 33482] [client 127.0.0.1:53110] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 55
[Thu Dec 09 08:12:51.655848 2021] [php:warn] [pid 33482] [client 127.0.0.1:53110] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 55
[Thu Dec 09 08:12:53.002596 2021] [php:warn] [pid 33482] [client 127.0.0.1:53110] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 55
[Thu Dec 09 08:15:04.123734 2021] [php:error] [pid 33486] [client 127.0.0.1:53134] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:05.355807 2021] [php:error] [pid 33484] [client 127.0.0.1:53140] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:06.243069 2021] [php:error] [pid 33485] [client 127.0.0.1:53144] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:07.020087 2021] [php:error] [pid 33955] [client 127.0.0.1:53148] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:07.736899 2021] [php:error] [pid 33482] [client 127.0.0.1:53152] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:08.406664 2021] [php:error] [pid 33483] [client 127.0.0.1:53156] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:09.867946 2021] [php:error] [pid 33486] [client 127.0.0.1:53160] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:10.938099 2021] [php:error] [pid 33484] [client 127.0.0.1:53164] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:15:11.815345 2021] [php:error] [pid 33485] [client 127.0.0.1:53168] PHP Fatal error:  Uncaught PDOException: SQLSTATE[23505]: Unique violation: 7 ERROR:  duplicate key value violates unique constraint "users_password_key"\nDETAIL:  Key (password)=(asdfasdfasdf) already exists. in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php:66\nStack trace:\n#0 /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php(66): PDOStatement->execute()\n#1 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(53): API\\models\\User->create()\n#2 /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php(63): API\\user\\reg()\n#3 /home/ninjainside/Projects/LOCAL/login-users/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 66
[Thu Dec 09 08:17:33.408972 2021] [php:warn] [pid 33955] [client 127.0.0.1:53182] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php on line 59
[Thu Dec 09 08:17:35.017092 2021] [php:warn] [pid 33955] [client 127.0.0.1:53182] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/user/reg.php on line 59
[Thu Dec 09 08:47:28.539679 2021] [php:warn] [pid 33483] [client 127.0.0.1:53734] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 65
[Thu Dec 09 08:47:30.243925 2021] [php:warn] [pid 33483] [client 127.0.0.1:53734] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 65
[Thu Dec 09 10:28:39.749603 2021] [php:warn] [pid 33482] [client 127.0.0.1:54082] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 111
[Thu Dec 09 10:28:40.946013 2021] [php:warn] [pid 33482] [client 127.0.0.1:54082] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 111
[Thu Dec 09 10:29:42.449573 2021] [php:warn] [pid 33955] [client 127.0.0.1:54104] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:21.178629 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:21.991329 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:22.620709 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:23.153898 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:23.658287 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:24.117464 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:24.617553 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:25.102797 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:30:25.544671 2021] [php:warn] [pid 33483] [client 127.0.0.1:54116] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 106
[Thu Dec 09 10:31:20.930003 2021] [php:warn] [pid 33486] [client 127.0.0.1:54142] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 115
[Thu Dec 09 10:31:21.766526 2021] [php:warn] [pid 33486] [client 127.0.0.1:54142] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 115
[Thu Dec 09 10:31:22.873959 2021] [php:warn] [pid 33486] [client 127.0.0.1:54142] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 115
[Thu Dec 09 10:31:23.377357 2021] [php:warn] [pid 33486] [client 127.0.0.1:54142] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 115
[Thu Dec 09 10:31:23.741466 2021] [php:warn] [pid 33486] [client 127.0.0.1:54142] PHP Warning:  Trying to access array offset on value of type bool in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 115
[Thu Dec 09 10:39:50.837424 2021] [php:warn] [pid 33482] [client 127.0.0.1:54238] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 122
[Thu Dec 09 10:39:51.953187 2021] [php:warn] [pid 33955] [client 127.0.0.1:54242] PHP Warning:  Array to string conversion in /home/ninjainside/Projects/LOCAL/login-users/api/models/User.php on line 122
[Fri Dec 10 23:48:06.144792 2021] [php:error] [pid 105516] [client 127.0.0.1:58026] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:48:07.326938 2021] [php:error] [pid 105517] [client 127.0.0.1:58028] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:48:08.199095 2021] [php:error] [pid 105518] [client 127.0.0.1:58030] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:48:08.873854 2021] [php:error] [pid 105519] [client 127.0.0.1:58032] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:48:09.605667 2021] [php:error] [pid 105526] [client 127.0.0.1:58034] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:48:10.235267 2021] [php:error] [pid 105515] [client 127.0.0.1:58036] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:49:08.653728 2021] [php:error] [pid 105516] [client 127.0.0.1:58042] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:49:09.579375 2021] [php:error] [pid 105517] [client 127.0.0.1:58044] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:49:10.412547 2021] [php:error] [pid 105518] [client 127.0.0.1:58046] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:36\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(36): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(79): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 36
[Fri Dec 10 23:50:36.857145 2021] [php:error] [pid 105519] [client 127.0.0.1:58060] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:29\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(29): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(81): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 29
[Fri Dec 10 23:50:37.670884 2021] [php:error] [pid 105526] [client 127.0.0.1:58062] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:29\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(29): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(81): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 29
[Fri Dec 10 23:50:38.444078 2021] [php:error] [pid 105515] [client 127.0.0.1:58064] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:29\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(29): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(81): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 29
[Fri Dec 10 23:50:39.077188 2021] [php:error] [pid 105516] [client 127.0.0.1:58066] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:29\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(29): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(81): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 29
[Fri Dec 10 23:51:25.368519 2021] [php:error] [pid 105517] [client 127.0.0.1:58072] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:31\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(31): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(83): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 31
[Fri Dec 10 23:51:26.209798 2021] [php:error] [pid 105518] [client 127.0.0.1:58074] PHP Fatal error:  Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php:31\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(31): array_filter()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php(83): API\\user\\auth()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/auth.php on line 31
[Sat Dec 11 00:17:17.442855 2021] [php:warn] [pid 105517] [client 127.0.0.1:58302] PHP Warning:  Undefined array key "Authorization" in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 18
[Sat Dec 11 00:17:18.470126 2021] [php:warn] [pid 105517] [client 127.0.0.1:58302] PHP Warning:  Undefined array key "Authorization" in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 18
[Sat Dec 11 01:11:30.867548 2021] [php:error] [pid 105516] [client 127.0.0.1:58598] PHP Fatal error:  Uncaught TypeError: strlen(): Argument #1 ($string) must be of type string, array given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php:30\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(30): strlen()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(49): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 30
[Sat Dec 11 01:11:32.025172 2021] [php:error] [pid 105526] [client 127.0.0.1:58600] PHP Fatal error:  Uncaught TypeError: strlen(): Argument #1 ($string) must be of type string, array given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php:30\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(30): strlen()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(49): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 30
[Sat Dec 11 01:11:32.998571 2021] [php:error] [pid 105519] [client 127.0.0.1:58602] PHP Fatal error:  Uncaught TypeError: strlen(): Argument #1 ($string) must be of type string, array given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php:30\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(30): strlen()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(49): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 30
[Sat Dec 11 01:11:50.638017 2021] [php:warn] [pid 105517] [client 127.0.0.1:58606] PHP Warning:  Undefined variable $errors in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 34
[Sat Dec 11 01:11:50.638084 2021] [php:error] [pid 105517] [client 127.0.0.1:58606] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php:34\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(34): count()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(49): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 34
[Sat Dec 11 01:11:51.539073 2021] [php:warn] [pid 105518] [client 127.0.0.1:58610] PHP Warning:  Undefined variable $errors in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 34
[Sat Dec 11 01:11:51.539148 2021] [php:error] [pid 105518] [client 127.0.0.1:58610] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php:34\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(34): count()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(49): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php on line 34
[Sat Dec 11 01:35:17.497823 2021] [php:notice] [pid 105518] [client 127.0.0.1:58830] PHP Notice:  Accessing static property API\\models\\Tip::$db_connect as non static in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:35:17.497883 2021] [php:warn] [pid 105518] [client 127.0.0.1:58830] PHP Warning:  Undefined property: API\\models\\Tip::$db_connect in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:35:17.497941 2021] [php:error] [pid 105518] [client 127.0.0.1:58830] PHP Fatal error:  Uncaught Error: Call to a member function prepare() on null in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:48\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:35:19.424419 2021] [php:notice] [pid 105515] [client 127.0.0.1:58834] PHP Notice:  Accessing static property API\\models\\Tip::$db_connect as non static in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:35:19.424473 2021] [php:warn] [pid 105515] [client 127.0.0.1:58834] PHP Warning:  Undefined property: API\\models\\Tip::$db_connect in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:35:19.424509 2021] [php:error] [pid 105515] [client 127.0.0.1:58834] PHP Fatal error:  Uncaught Error: Call to a member function prepare() on null in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:48\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:37:48.154529 2021] [php:notice] [pid 105516] [client 127.0.0.1:58862] PHP Notice:  Accessing static property API\\models\\Tip::$db_connect as non static in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:37:48.154585 2021] [php:warn] [pid 105516] [client 127.0.0.1:58862] PHP Warning:  Undefined property: API\\models\\Tip::$db_connect in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:37:48.154634 2021] [php:error] [pid 105516] [client 127.0.0.1:58862] PHP Fatal error:  Uncaught Error: Call to a member function prepare() on null in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:48\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(51): API\\models\\Tip->read()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(55): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:37:49.343762 2021] [php:notice] [pid 105526] [client 127.0.0.1:58866] PHP Notice:  Accessing static property API\\models\\Tip::$db_connect as non static in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:37:49.343825 2021] [php:warn] [pid 105526] [client 127.0.0.1:58866] PHP Warning:  Undefined property: API\\models\\Tip::$db_connect in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:37:49.343874 2021] [php:error] [pid 105526] [client 127.0.0.1:58866] PHP Fatal error:  Uncaught Error: Call to a member function prepare() on null in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:48\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(51): API\\models\\Tip->read()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(55): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:14.233864 2021] [php:notice] [pid 105519] [client 127.0.0.1:58878] PHP Notice:  Accessing static property API\\models\\Tip::$db_connect as non static in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:14.233945 2021] [php:warn] [pid 105519] [client 127.0.0.1:58878] PHP Warning:  Undefined property: API\\models\\Tip::$db_connect in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:14.233988 2021] [php:error] [pid 105519] [client 127.0.0.1:58878] PHP Fatal error:  Uncaught Error: Call to a member function prepare() on null in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:48\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:16.427432 2021] [php:notice] [pid 105517] [client 127.0.0.1:58882] PHP Notice:  Accessing static property API\\models\\Tip::$db_connect as non static in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:16.427487 2021] [php:warn] [pid 105517] [client 127.0.0.1:58882] PHP Warning:  Undefined property: API\\models\\Tip::$db_connect in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:16.427526 2021] [php:error] [pid 105517] [client 127.0.0.1:58882] PHP Fatal error:  Uncaught Error: Call to a member function prepare() on null in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:48\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 48
[Sat Dec 11 01:39:57.237409 2021] [php:error] [pid 105518] [client 127.0.0.1:58890] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 2:             SELECT user.fullname tip.id tip.content FROM use...\n                               ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:58\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(58): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 58
[Sat Dec 11 01:39:58.493599 2021] [php:error] [pid 105515] [client 127.0.0.1:58894] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 2:             SELECT user.fullname tip.id tip.content FROM use...\n                               ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:58\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(58): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 58
[Sat Dec 11 01:40:26.089537 2021] [php:error] [pid 105516] [client 127.0.0.1:58900] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 2:             SELECT user.fullname, tip.id, tip.content FROM u...\n                               ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:58\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(58): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 58
[Sat Dec 11 01:40:26.962165 2021] [php:error] [pid 105526] [client 127.0.0.1:58904] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 2:             SELECT user.fullname, tip.id, tip.content FROM u...\n                               ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:58\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(58): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 58
[Sat Dec 11 01:41:31.887208 2021] [php:error] [pid 105519] [client 127.0.0.1:58912] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 2:             SELECT user.fullname, tip.id, tip.content FROM u...\n                               ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:58\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(58): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 58
[Sat Dec 11 01:41:32.807418 2021] [php:error] [pid 105517] [client 127.0.0.1:58916] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 2:             SELECT user.fullname, tip.id, tip.content FROM u...\n                               ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:58\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(58): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 58
[Sat Dec 11 01:42:10.876917 2021] [php:error] [pid 105518] [client 127.0.0.1:58924] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 5:             WHERE user.token = 0b963aebdad0d5d0c5bb7d7cc30c3...\n                              ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:42:11.574288 2021] [php:error] [pid 105515] [client 127.0.0.1:58928] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "."\nLINE 5:             WHERE user.token = 0b963aebdad0d5d0c5bb7d7cc30c3...\n                              ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:42:33.541562 2021] [php:error] [pid 105516] [client 127.0.0.1:58934] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "b963aebdad0d5d0c5bb7d7cc30c30db"\nLINE 5:             WHERE u.token = 0b963aebdad0d5d0c5bb7d7cc30c30db\n                                     ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:42:34.424119 2021] [php:error] [pid 105526] [client 127.0.0.1:58938] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "b963aebdad0d5d0c5bb7d7cc30c30db"\nLINE 5:             WHERE u.token = 0b963aebdad0d5d0c5bb7d7cc30c30db\n                                     ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:43:10.298396 2021] [php:error] [pid 105519] [client 127.0.0.1:58962] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  invalid reference to FROM-clause entry for table "users"\nLINE 4:             ON users.id = tips.user_id\n                       ^\nHINT:  Perhaps you meant to reference the table alias "u". in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:43:11.165117 2021] [php:error] [pid 105517] [client 127.0.0.1:58966] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  invalid reference to FROM-clause entry for table "users"\nLINE 4:             ON users.id = tips.user_id\n                       ^\nHINT:  Perhaps you meant to reference the table alias "u". in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:43:47.137611 2021] [php:error] [pid 105518] [client 127.0.0.1:58976] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  invalid reference to FROM-clause entry for table "tips"\nLINE 4:             ON tips.id = users.user_id\n                       ^\nHINT:  Perhaps you meant to reference the table alias "t". in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:43:47.813433 2021] [php:error] [pid 105515] [client 127.0.0.1:58980] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  invalid reference to FROM-clause entry for table "tips"\nLINE 4:             ON tips.id = users.user_id\n                       ^\nHINT:  Perhaps you meant to reference the table alias "t". in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:59\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(59): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 59
[Sat Dec 11 01:45:10.378518 2021] [php:error] [pid 105516] [client 127.0.0.1:58994] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42703]: Undefined column: 7 ERROR:  column users.user_id does not exist\nLINE 5:             ON tips.id = users.user_id\n                                 ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:60\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(60): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 60
[Sat Dec 11 01:47:53.039229 2021] [php:error] [pid 105526] [client 127.0.0.1:59092] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  missing FROM-clause entry for table "u"\nLINE 6:             WHERE u.token = '0b963aebdad0d5d0c5bb7d7cc30c30d...\n                          ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:60\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(60): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 60
[Sat Dec 11 01:47:54.461793 2021] [php:error] [pid 105518] [client 127.0.0.1:59096] PHP Fatal error:  Uncaught PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR:  missing FROM-clause entry for table "u"\nLINE 6:             WHERE u.token = '0b963aebdad0d5d0c5bb7d7cc30c30d...\n                          ^ in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php:60\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php(60): PDOStatement->execute()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(50): API\\models\\Tip->read()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/api/user/read.php(54): API\\user\\read()\n#3 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#4 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/models/Tip.php on line 60
[Sat Dec 11 02:44:09.005177 2021] [php:warn] [pid 105526] [client 127.0.0.1:59620] PHP Warning:  Undefined variable $invalid_errors in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:09.005224 2021] [php:error] [pid 105526] [client 127.0.0.1:59620] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php:38\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(38): count()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(80): API\\user\\reg()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:10.330082 2021] [php:warn] [pid 105518] [client 127.0.0.1:59622] PHP Warning:  Undefined variable $invalid_errors in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:10.330143 2021] [php:error] [pid 105518] [client 127.0.0.1:59622] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php:38\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(38): count()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(80): API\\user\\reg()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:11.130539 2021] [php:warn] [pid 105516] [client 127.0.0.1:59624] PHP Warning:  Undefined variable $invalid_errors in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:11.130609 2021] [php:error] [pid 105516] [client 127.0.0.1:59624] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php:38\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(38): count()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(80): API\\user\\reg()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:11.640843 2021] [php:warn] [pid 105519] [client 127.0.0.1:59626] PHP Warning:  Undefined variable $invalid_errors in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
[Sat Dec 11 02:44:11.640906 2021] [php:error] [pid 105519] [client 127.0.0.1:59626] PHP Fatal error:  Uncaught TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php:38\nStack trace:\n#0 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(38): count()\n#1 /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php(80): API\\user\\reg()\n#2 /home/ninjainside/Projects/GIT/test-auth-php/index.php(11): require_once('...')\n#3 {main}\n  thrown in /home/ninjainside/Projects/GIT/test-auth-php/api/user/reg.php on line 38
